flink-components/src/test/scala/pl/touk/nussknacker/sample/SampleComponentProviderTest.scala

```diff
@@ -20,7 +20,7 @@ class SampleComponentProviderTest extends Matchers with ValidatedValuesDetailedM
       .streaming("test scenario")
       .source("custom-source-node-name", TestScenarioRunner.testDataSource)
       .enricher("component-provider-service-node-name", "output", "randomString", "length" -> "#input")
-      .processorEnd("end", TestScenarioRunner.testResultService, "value" -> "#output")
+      .emptySink("end", TestScenarioRunner.testResultSink, "value" -> "#output")
```

Comment:
I'm not sure about this change from `processorEnd` to `emptySink` but otherwise a 
`The Validated on which validValue was invoked was defined as Invalid(NonEmptyList(MissingService(sink,end)))` error is thrown

Reply:
its ok, it was done analogically in main repo tests after TestRunner refactor